test/test_restful.py

```diff
@@ -276,6 +277,33 @@ def initthread():
     return True
 
 
+def launch_rust_cloudagent():
+    """Start up the cloud agent"""
+    global agent_process, script_env, FORK_ARGS
+    if agent_process is None:
+        agent_process = subprocess.Popen("/.cargo/bin/keylime_agent",
```

Comment:
Do you think that the rust agent will be in the root? Is because is inside a container?

Reply:
I realized I pushed a broken path, which I've now fixed. It should be correct now.